### DISCUSSION TITLE
Where can I find a list of the supported front matter features?

### DISCUSSION THREAD

**Discussion by vbd** [squidfunk/mkdocs-material#4872] [Q&A] [answered]
Is there one and where can I find it? 
Thanks in advance.

**Comment by squidfunk** [comment, marked as answer]
Our documentation mentions those features at different locations, i.e., where it makes sense. There's currently no list that lists all of them, but for starters, you should check the __Usage__ sections in our documentation:

- [Reference](https://squidfunk.github.io/mkdocs-material/reference/#usage): Basic features like `title`, `description`
- [Setting up navigation](https://squidfunk.github.io/mkdocs-material/setup/setting-up-navigation/#usage): Hiding sidebars
- [Setting up site search](https://squidfunk.github.io/mkdocs-material/setup/setting-up-site-search/#usage): Search tuning
- [Setting up site analytics](https://squidfunk.github.io/mkdocs-material/setup/setting-up-site-analytics/#usage): Hiding feedback widget
- [Setting up a blog](https://squidfunk.github.io/mkdocs-material/setup/setting-up-a-blog/#usage): Blog features like `date` or `draft`
- [Setting up tags](https://squidfunk.github.io/mkdocs-material/setup/setting-up-tags/#usage): Tags features
- [Setting up the footer](https://squidfunk.github.io/mkdocs-material/setup/setting-up-the-footer/#usage): Hiding the footer links

**Reply by vbd** [reply]
Thank you this will help me additional. I already found some of the features by reading trough the documentation. I'm looking for the `some_url` feature and what is it used for. I'm not sure if it is used for the canoncial url of the page or not.
